src/pages/chanyoung/Main/components/feeds/Comment.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import CommentWnd from './CommentWnd';
+import CommentForm from './CommentForm';
+
+class Comment extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      commentsInfo: {
+        list: [],
+        nextKey: null,
+      },
+    };
+  }
+
+  componentDidMount() {
+    this.initComment();
+  }
+
+  initComment = () => {
+    fetch('http://localhost:3000/chanyoung/data/commentData.json')
+      .then(res => res.json())
+      .then(data => {
+        this.setState(() => ({
+          commentsInfo: {
+            list: data,
+            nextKey: data.length + 1,
+          },
+        }));
+      });
+  };
+
+  addCommnet = e => {
+    e.preventDefault();
+    const { nextKey } = this.state.commentsInfo;
+    const newComment = {
+      key: nextKey,
+      name: '정찬영',
+      comment: e.target[0].value,
+    };
+
+    this.setState(({ commentsInfo: { list } }) => {
+      return {
+        commentsInfo: {
+          list: [...list, newComment],
+          nextKey: nextKey + 1,
+        },
+      };
+    });
+    e.target[0].value = '';
+  };
+
+  deleteComment = key => {
+    this.setState(({ commentsInfo: { list, nextKey } }) => ({
+      commentsInfo: {
+        list: list.filter(e => key !== e.key),
+        nextKey: nextKey,
```

Comment:
```suggestion
        nextKey
```
key와 value가 이름이 같으면 축약 가능합니다

Reply:
놓쳤던 부분이네요 수정했습니다!